src/foraging_gui/Foraging.py

```diff
@@ -4661,13 +4675,18 @@ def _generate_upload_manifest(self, session: Session):
             :param session: session to use to create upload manifest
         '''
 
+        # skip manifest generation for test mouse
         if self.behavior_session_model.subject in ['0','1','2','3','4','5','6','7','8','9','10']:
             logging.info('Skipping upload manifest, because this is the test mouse')
             return
-
+        # skip manifest generation if automatic upload is disabled
         if not self.Settings['AutomaticUpload']:
```

Comment:
Why not use the "AutomaticUpload" setting to turn off manifest generation instead?

Reply:
For the ephys rig, we are also using the manifest generation for behavior sessions without the ephys recording.

Reply:
Why are you not uploading ephys sessions? We need a clear plan for how we are going to upload and process ephys data, and how we will generate ephys specific metadata. We had previously discussed with @KanghoonJ that ephys specific metadata should be handled by a ephys specific software base.